tur/topiary/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://topiary.tweag.io/
+TERMUX_PKG_DESCRIPTION="The universal code formatter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+TERMUX_PKG_VERSION="0.6.1"
+TERMUX_PKG_SRCURL=https://github.com/tweag/topiary/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=e2bbac9cb46a3743cc41ca55245026580308722242c9df84bc0ef3cbb989aa81
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE=latest-release-tag
+
+termux_step_pre_configure() {
+	# clash with rust host build
+	# causes 32bit builds to fail if set
+	unset CFLAGS
+	# do not pin the version of the rust toolchain
+	rm -f rust-toolchain.toml
```

Comment:
That comment and `rm` command was specific to the `nerdfix` package.
`topiary` doesn't pin the rust version it uses, so this shouldn't be necessary.